infra-as-code/bicep/modules/policy/assignments/alzDefaults/alzDefaultPolicyAssignments.bicep

```
@@ -1,6 +1,25 @@
 metadata name = 'ALZ Bicep - ALZ Default Policy Assignments'
 metadata description = 'This module will assign the ALZ Default Policy Assignments to the ALZ Management Group hierarchy'
 
+type policyAssignmentSovereigntyGlobalOptionsType = ({
+  @sys.description('Switch to enable/disable deployment of the Sovereignty Baseline - Global Policies Assignment to the intermediate root management group.')
+  parTopLevelSovereigntyGlobalPoliciesEnable: bool
+
+  @sys.description('The list of locations that your organization can use to restrict deploying resources to. If left empty, only the deployment location will be allowed.')
+  parListOfAllowedLocations: string[]
+})
+
+type policyAssignmentSovereigntyConfidentialOptionsType = ({
+  @sys.description('The list of Azure resource types approved for usage, which is the set of resource types that have a SKU backed by Azure Confidential Computing or resource types that do not process customer data. Leave empty to allow all relevant resource types.')
+  parAllowedResourceTypes: string[]
+
+  @sys.description('The list of locations that your organization can use to restrict deploying resources to. If left empty, only the deployment location will be allowed.')
+  parListOfAllowedLocations: string[]
+
+  @sys.description('The list of VM SKUs approved approved for usage, which is the set of SKUs backed by Azure Confidential Computing. Leave empty to allow all relevant SKUs.')
+  parAllowedVirtualMachineSKUs: string[]
+})
```

```suggestion
type policyAssignmentSovereigntyConfidentialOptionsType = {
  @sys.description('The list of Azure resource types approved for usage, which is the set of resource types that have a SKU backed by Azure Confidential Computing or resource types that do not process customer data. Leave empty to allow all relevant resource types.')
  parAllowedResourceTypes: string[]

  @sys.description('The list of locations that your organization can use to restrict deploying resources to. If left empty, only the deployment location will be allowed.')
  parListOfAllowedLocations: string[]

  @sys.description('The list of VM SKUs approved approved for usage, which is the set of SKUs backed by Azure Confidential Computing. Leave empty to allow all relevant SKUs.')
  parAllowedVirtualMachineSKUs: string[]
}
```
dont think the parentheses are needed

---

You're right, confirmed with docs and testing.